### DISCUSSION TITLE
Compatibility with OneDrive

### DISCUSSION THREAD

**Discussion by rdaaron** [kangyu-california/PersistentWindows#249] [General]
Greetings. First of, thanks SOOO much for this product!!

I have an Office 365 subscription and use OneDrive where many of my Word and Excel files are kept locally on my disk (and sync'ed to my other devices). Didn't really realized this until now, but O365 uses the URL instead of the fully qualified OS file name. PW has trouble with this.

For Word files, it will open a blank document instead.

For Excel, I get a dialog box saying the file can't be found. 

(strange the two apps present different issues but anyway).

In each app under Info, there's a "Copy path" and "Copy local path" button. Here is the output for a Word and Excel document. 

https://d.docs.live.net/33c14928c6e06498/PINN/Writing/PINN%20Presentation.docx
C:\Users\Rob\OneDrive\PINN\Writing\PINN Presentation.docx

https://d.docs.live.net/33c14928c6e06498/PINN/Writing/PINN%20Models%20v2.xlsx
C:\Users\Rob\OneDrive\PINN\Writing\PINN Models v2.xlsx

If the solution is to get O365 to use the local path version, that could be a problem. I have many links from one Word document to another. Those links are the URL version which works well whatever computer I'm using (I switch between my workstation and laptop). If it were local path, it MAY be that the local path would be used instead moving forward (which would break things). I don't know if having O365 switch to using local path is an option but just wanted to say, if it is, for me at least, it would cause trouble. 

I'm surprised no one has run into this before. No OneDrive users out there?? :-)

Many thanks,
Rob A

**Comment by kangyu-california** [comment]
It is a known issue that PW does not restore Microsoft Store App window from disk.
PW captures process command line to disk by running this command
 "wmic.exe process get commandline,processid /format:csv"
However the command line does not work for Microsoft Store App somehow.

PW users are advised to manually launch the App windows and open corresponding files, and then run PW Restore From Disk again to precisely restore the window location.

**Comment by kangyu-california** [comment]
Created issue #250 for tracking

**Comment by kangyu-california** [comment]
Microsoft Store App does not seem to support file path / URL on command line.

You may try the latest 5.45 release.
